defaults/main.yml

```diff
@@ -3,7 +3,7 @@
 # To connect Linux machines you need to setup a key
 # https://login.tailscale.com/admin/settings/authkeys
 tailscale_authkey: "{{ lookup('env','TAILSCALE_AUTHKEY') }}"
-tailscale_args: '--shields-up'
+tailscale_args: '--accept-routes=true'
```

Comment:
This is not the right place ... change the args from the use case in role-base-quiknode ...

Reply:
thanks ...closing this one